docs/GettingStarted.md

```diff
@@ -0,0 +1,80 @@
+# Getting started
+
+All you'll a [Java 8 JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html) and the Hail source code: either clone the [Hail repository](https://github.com/broadinstitute/hail) using [Git](https://git-scm.com/) with
+```
+git clone https://github.com/broadinstitute/hail.git
+```
+or download the source code directly from [here](https://github.com/broadinstitute/hail/archive/master.zip), unzip, and rename the enclosing `hail-master` directory to `hail`. The following commands are relative to this `hail` directory.
```

Comment:
> The following commands are relative to this `hail` directory.

As written, this doesn't sound like it refers to the hail directory of the clone (which you don't even mention).  I'd prefer something like:

```
$ git clone ...
$ cd hail
```

The commands below are relative to the `hail` directory.

You can also download the source code directly from [here](...).